frontend/packages/ceph-storage-plugin/src/components/create-storage-system/footer.tsx

```diff
@@ -86,72 +87,93 @@ const canJumpToNextStep = (name: string, state: WizardState, t: TFunction) => {
     case StepsName(t)[Steps.CapacityAndNodes]:
       return nodes.length >= MINIMUM_NODES && capacity;
     case StepsName(t)[Steps.SecurityAndNetwork]:
-      return encryption.hasHandled && hasConfiguredNetwork && kms.hasHandled;
+      return encryption.hasHandled && kms.hasHandled && hasConfiguredNetwork;
+    case StepsName(t)[Steps.Security]:
+      return encryption.hasHandled && kms.hasHandled;
     case StepsName(t)[Steps.ReviewAndCreate]:
       return true;
     default:
       return false;
   }
 };
 
-const getPayloads = (stepName: string, state: WizardState, hasOCS: boolean, t: TFunction) => {
-  const { backingStorage, createStorageClass, storageClass, connectionDetails } = state;
-  const { externalStorage, deployment, type } = backingStorage;
-
+const handleBackingStorageNext = async (
+  backingStorage: WizardState['backingStorage'],
+  handleError: (err: string) => void,
+  storageSystems: StorageSystemKind[] = [],
+  moveToNextStep: () => void,
+) => {
+  const { externalStorage, type } = backingStorage;
+  const { model, displayName } = getExternalStorage(externalStorage) || {};
+  const isSSPresent = storageSystems.find((ss) => ss.kind === model.kind);
   const isRhcs = externalStorage === OCSServiceModel.kind;
```

Comment:
```suggestion
  const isSSPresent = storageSystems.find((ss) => ss?.kind === model?.kind);
  const isRhcs = externalStorage === OCSServiceModel?.kind;
```

Reply:
Actually it should be ss.spec.kind. Thanks for pointing.

Reply:
Fixed